packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -297,6 +297,11 @@ class WebFs {
 class BuildDaemonCreator {
   const BuildDaemonCreator();
 
+  // TODO(jonahwilliams): find a way to get build checks working for flutter for web.
```

Comment:
I thought this was a non-goal? Given that it is shipped with the sdk and we invalidate the build if the SDK version changes.